metaflow/plugins/kfp/kfp.py

```diff
@@ -1162,6 +1295,7 @@ def _create_metaflow_step_op(
         step_variables: StepVariables,
         flow_variables: FlowVariables,
         metaflow_configs: Dict[str, str],
+        flow_parameters: Dict,
         flow_parameters_json: str,
```

Comment:
It looks like `flow_parameters_json` is always set to `"{{workflow.parameters}}"` whenever `_create_metaflow_step_op` function is called. Can we remove `flow_parameters_json` from parameter list of this function and use it as a string literal?

Reply:
nice, fixed with https://github.com/zillow/metaflow/blob/199c3320fa4c940b308fd9267e89395709f5952b/metaflow/plugins/kfp/kfp.py#L92